### ISSUE TITLE
`ionic cordova run ios` fails with v6.0.7 - ok with v6.0.6

### ISSUE THREAD

**Issue by tsimbalar** [dpa99c/cordova-plugin-firebasex#125]
# Bug report

**Current behavior:**

On an `ionic 4` project, running `ionic cordova run ios` shows the following output : 

```
ionic cordova run ios
[INFO] Hardware device(s) found for ios. Using --device.
> ng run app:ionic-cordova-build --platform=ios

chunk {0} 0.js, 0.js.map () 14.6 kB  [rendered]
chunk {1} 1.js, 1.js.map () 16 kB  [rendered]
// snipped ...
chunk {tap-click-b300ec79-js} tap-click-b300ec79-js.js, tap-click-b300ec79-js.js.map (tap-click-b300ec79-js) 6.32 kB  [rendered]
chunk {vendor} vendor.js, vendor.js.map (vendor) 5.6 MB [initial] [rendered]
Date: 2019-09-02T08:53:17.419Z - Hash: 9e0352e04a0172976a5e - Time: 40284ms
> cordova build ios --device
Preparing Firebase on iOS
> native-run ios --app platforms/ios/build/device/Perspectives.ipa --device
[native-run] ERR_UNKNOWN: Path 'platforms/ios/build/device/Perspectives.ipa' not found
[ERROR] An error occurred while running subprocess native-run.

        native-run ios --app platforms/ios/build/device/Perspectives.ipa --device exited with exit code 1.

        Re-running this command with the --verbose flag may provide more information.
```

It fails just after `Preparing Firebase on iOS` . 

When I do the same thing with version `v6.0.6` of the plugin, I don't get that error.

**Expected behavior:**

It should continue and launch the ios run

**Steps to reproduce:**
run `ionic cordova run ios`

**Environment information**

- Cordova CLI version 
	- `9.0.0 (cordova-lib@9.0.1)`
- Cordova platform version
	- `cordova platform ls`

```
Installed platforms:
  ios 5.0.1
Available platforms:
  android ^8.0.0
  browser ^6.0.0
  electron ^1.0.0
  osx ^5.0.0
  windows ^7.0.0
```

- Plugins & versions installed in project (including this plugin)
    - `cordova plugin ls` (after rollbacking to v6.0.6)

```
cordova-plugin-androidx 1.0.2 "cordova-plugin-androidx"
cordova-plugin-androidx-adapter 1.1.0 "cordova-plugin-androidx-adapter"
cordova-plugin-app-version 0.1.9 "AppVersion"
cordova-plugin-badge 0.8.8 "Badge"
cordova-plugin-device 2.0.2 "Device"
cordova-plugin-firebasex 6.0.6 "Google Firebase Plugin"
cordova-plugin-inappbrowser 3.0.0 "InAppBrowser"
cordova-plugin-ionic-keyboard 2.1.3 "cordova-plugin-ionic-keyboard"
cordova-plugin-ionic-webview 4.1.1 "cordova-plugin-ionic-webview"
cordova-plugin-network-information 2.0.1 "Network Information"
cordova-plugin-pincheck 0.0.6 "PinCheck"
cordova-plugin-splashscreen 5.0.2 "Splashscreen"
cordova-plugin-statusbar 2.4.2 "StatusBar"
cordova-plugin-whitelist 1.3.3 "Whitelist"
```

- Dev machine OS and version, e.g.
    - OSX
        - `sw_vers`
```
ProductName:	Mac OS X
ProductVersion:	10.14.6
BuildVersion:	18G84
```

_iOS build issue:_
- Node JS version
    - `node -v` : `v10.16.3`
- XCode version

**Other information:**

This issue is opened also, and redirected me to here : 

https://github.com/ionic-team/native-run/issues/41

It seems also related to https://github.com/dpa99c/cordova-plugin-firebasex/issues/5#issuecomment-526342898

Thanks so much for your work !

**Comment by tsimbalar** [comment]
This looks similar to https://github.com/dpa99c/cordova-plugin-firebasex/issues/123, but not 100% it's a duplicate.

Feel free to close if you think it is and want to track them from a single place.

**Comment by tsimbalar** [comment]
Oh it seems to be the same issue as fixed by https://github.com/dpa99c/cordova-plugin-firebasex/pull/121

**Comment by dpa99c** [comment]
I've just published `cordova-plugin-firebasex@6.1.0` which contains the fix in #121 so this should resolve your issue. Please let me know if it doesn't.

**Comment by tsimbalar** [comment]
It does fix it, yes ! 

Thanks for your reactivity and support :)
